src/main/java/models/Admin.java

```diff
@@ -0,0 +1,15 @@
+package models;
+
+public class Admin extends User{
```

Comment:
check item 5 in task and add missing method

Reply:
I have this method "uniqueAbility". I extends it from Class User and then @Override it.

Reply:
it is not about overriding, it is about having specific method for each class. 
In User class we have only one common method - printRole, which can be (but not necessary) overridden in subclasses. And for each subclass we need to have specific method (getTicket or checkTicket)

Reply:
+